#!/usr/bin/env python
PACKAGE = "performance_tests"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("rate_param", int_t, 0, "loop rate of publish", 50,  0, 100000)

exit(gen.generate(PACKAGE, "py_pub_node", "DynRate"))